api/api_net.go

```diff
@@ -64,6 +64,9 @@ type Net interface {
 
 	// ID returns peerID of libp2p node backing this API
 	ID(context.Context) (peer.ID, error) //perm:read
+	// PrivKey returns libp2p host's private key.
+	// This is temporal experimental function that must be removed in the future.
```

Comment:
```suggestion
	// FIXME: This is temporal experimental function that must be removed in the future.
```
That way is easier to spot in the future? Exposing the private key is always a bit frightening :)